kong/plugins/hmac-auth/access.lua

```diff
@@ -79,30 +84,31 @@ local function create_hash(request, hmac_params, headers)
       signing_string = signing_string.."\n"
     end
   end
+
   return ngx_sha1(hmac_params.secret, signing_string)
 end
 
-local function is_digest_equal(digest_1, digest_2)
+local function is_signature_equal(signature_1, signature_2)
```

Comment:
what are we changing this function name and variables? this dirties the commit history with no real value, and decreases the atomicity of this commit.